samples/subsys/ipc/ipm_mhu_dual_core/README.rst

```diff
@@ -0,0 +1,48 @@
+.. _ipm_mhu_dual_core:
+
+MHU Dual Core
+###########
```

Comment:
The heading underline must be at least as long as the title, so add a couple more ``##``